tests/py/test_api_multi_gpu.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import trtorch
+import torch
+import torchvision.models as models
+
+from model_test_case import ModelTestCase
+
+gpu_id = 1
+class TestCompile(ModelTestCase):
```

Comment:
Why are we duplicating tests for logging and op support?

Reply:
Intial plan was to run either test_api.py or test_api_multi_gpu.py via Bazel BUILD file. Removed it now